### PR TITLE
Remove requirement for DC key_id and team_id to be secrets

### PR DESCRIPTION
**Pull request by sethvargo**
I've done a lot of reading and people generally don't consider these secrets.

### CODE REVIEW

File: migrations/000023_dc_less_secrets.up.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE AuthorizedApp RENAME COLUMN devicecheck_team_id_secret TO devicecheck_team_id;
```
Comment (sethvargo) [inline migrations/000023_dc_less_secrets.up.sql:17]: I don't think we have any in production. I'm not sure how to write a truly backwards-compat migration in pure SQL...

Comment (mikehelmick) [reply migrations/000023_dc_less_secrets.up.sql:17]: it’s a straight rename.

the field is being interpreted differently for sure, but I think this is OK.

**Pull request merged**